tasks/marketplacesvm-push-disk-images/marketplacesvm-push-disk-images.yaml

```diff
@@ -140,4 +144,9 @@ spec:
         pushsource-ls "staged:${BASE_DIR}"
 
         # Process the push
-        marketplacesvm_push_wrapper --debug --source "${BASE_DIR}" --starmap-file "$STARMAP_MAPPING_FILE"
+        pre_push="$(params.prePush)"
+        add_args=""
+        if [[ "${pre_push,,}" == "true" ]]; then
```

Comment:
What are the `,,` for?

Reply:
To put the value in lowercase so it can do the string comparison without being case sensitive

Reply:
I see. I'd say it's probably not needed. We have a number of options like this and we never allow uppercase. But this brings me to another thing: The new option should be added to the schema file. That will also ensure that it's the right format - if you specify it as boolean, then reading it with `jq` should always return the boolean "true" or "false".

Reply:
Adjusted!